src/modules/fields.js

```diff
@@ -0,0 +1,20 @@
+export const cleanList = () => {
+  const toDoList = document.querySelector('.todo-list');
+  while (toDoList.firstChild) {
+    toDoList.removeChild(toDoList.lastChild);
+  }
+};
+
+export const clearAll = document.querySelector('.todo-clear-all-completed');
+
+export const selectAll = document.querySelector('.select-all');
+
+export const clickPlus = document.querySelector('.image-plus');
+
+export const newItem = document.getElementById('newItem');
+
+export const toDoList = document.querySelector('.todo-list');
+
+export default class DataClass {
```

Comment:
you are using to many exports 
you can make only one export and include all the function in it 
its easier and faster 
example 

export { clear All, selectAll , .....}